Makefile

```diff
@@ -49,7 +50,8 @@ login_%: _check_venv
 
 ## Run 'molecule test --destroy=never' (run 'make destroy' to destroy containers)
 test: _check_venv
-	@.venv_ansible$(ANSIBLE_INSTALL_VERSION)/bin/molecule test --destroy=never
+	@.venv_ansible$(ANSIBLE_INSTALL_VERSION)/bin/molecule test --scenario-name java8 --destroy=never
+	@.venv_ansible$(ANSIBLE_INSTALL_VERSION)/bin/molecule test --scenario-name java11 --destroy=never
```

Comment:
Is it possible to run these in parallel?  With `nohup` or `&` or something?

Reply:
I will check if you can run multiple scenarios with molecule, but if not I would rather stay away from `nohup` end similar.